apps/css-workshop/src/pages/menu.astro

```diff
@@ -38,239 +39,134 @@ import Icon_ from '../components/Icon.astro';
   <h2>Default</h2>
   <section id='demo-default'>
     <ul class='iui-menu' role='menu'>
-      <li
-        class='iui-list-item'
-        data-iui-actionable='true'
-        role='menuitem'
-        tabindex='0'
-        id='test-menu-1'
-      >
-        <span class='iui-list-item-content'>Option 1</span>
-      </li>
-      <li class='iui-list-item' data-iui-actionable='true' role='menuitem' tabindex='0'>
-        <span class='iui-list-item-content'>Option 2</span>
-      </li>
-      <li class='iui-list-item iui-disabled' data-iui-actionable='true'>
-        <span class='iui-list-item-content'>Option 3</span>
-      </li>
+      <Menu_ tabindex='0' id='test-menu-1' role='menuitem'>Option 1</Menu_>
+      <Menu_ tabindex='0' role='menuitem'>Option 2</Menu_>
+      <Menu_ disabled>Option 3</Menu_>
     </ul>
 
     <br />
 
     <ul class='iui-menu' role='menu'>
-      <li
-        class='iui-list-item iui-active'
-        data-iui-actionable='true'
-        role='menuitem'
-        aria-checked='true'
-        tabindex='0'
-      >
-        <span class='iui-list-item-content'>Option 1</span>
-        <svg-checkmark class='iui-list-item-icon' aria-hidden='true'></svg-checkmark>
-      </li>
-      <li class='iui-list-item' data-iui-actionable='true' role='menuitem' tabindex='0'>
-        <span class='iui-list-item-content'>Option 2</span>
-      </li>
-      <li
-        class='iui-list-item iui-active'
-        data-iui-actionable='true'
-        role='menuitem'
-        aria-checked='true'
-        tabindex='0'
-      >
-        <span class='iui-list-item-content'>Option 3</span>
-        <svg-checkmark class='iui-list-item-icon' aria-hidden='true'></svg-checkmark>
-      </li>
+      <Menu_ isActive tabindex='0' role='menuitem' checked>
```

Comment:
`checked` does not make sense here, was it accidentally added?

Reply:
Right, some `checked`'s were mistakenly placed. I have removed the redundant ones.